src/StreamFactoryInterface.php

```diff
@@ -9,15 +9,11 @@
     /**
      * Create a new stream from a resource or a string.
      *
-     * If the argument is a resource, it MUST be readable and SHOULD be seekable. It MAY be writable.
-     * If the argument is a string, a temporary resource will be created that is writable and seekable.
-     * 
-     * If the arugment is a string it will be interpided as the content of the stream. File names or 
-     * file paths are not supported. 
+     * String content will be used as the body of the stream. Files paths are not supported.
```

Comment:
> We are designing an interface that library authors should rely on. If someone (or we) creates an utility library, I would consider that as a failure for PSR-17.

IMO, that line of argumentation is problematic, because that implies you think there is one correct library design - which, if that is the case, where is the need for interoperability? If one design will satisfy everyone?

I don't think it will.

If there's a utility library that neatly implements a stable, tested, reliable utility library, that works for all implementations of PSR-17, how is that a failure exactly?

If there are multiple such libraries competing, all working for PSR-17, that creates room for developers to compete for creation of the best design and implementation. I don't want PSR-17 to stand in the way of that. If it dictates architecture, it will. If the design is dictated by the interfaces, why would anyone even care if there are one or more implementations? They would all work precisely the same way.

IMO, we should not be designing the solution, but the abstraction on which interchangible solutions can be built.